orcid-core/src/main/java/org/orcid/core/security/visibility/aop/OrcidApiAuthorizationSecurityAspect.java

```diff
@@ -318,13 +321,14 @@ private boolean isUpdateOrReadScope(ScopePathType requiredScope) {
             return false;
         }
     }
-
-    private boolean hasScopeEnabled(String clientId, String userName, String scope, String equivalentScope) {
+    
+    @Deprecated
+    public boolean hasScopeEnabled(String clientId, String userName, String scope, String equivalentScope) {
```

Comment:
Make it public so we can do a round of unit testing on it, but we should not use it anywhere outside this class.